core/storage/base_model/gae_models.py

```diff
@@ -773,14 +773,13 @@ def delete_multi(cls, entity_ids, committer_id,
                 snapshot = model.compute_snapshot()
                 snapshot_id = model.get_snapshot_id(model.id, model.version)
 
-                snapshot_metadata_models.append(model.SNAPSHOT_METADATA_CLASS(  # pylint: disable=not-callable
-                    id=snapshot_id,
-                    committer_id=committer_id,
-                    commit_type=cls._COMMIT_TYPE_DELETE,
-                    commit_message=commit_message,
-                    commit_cmds=commit_cmds))
-                snapshot_content_models.append(model.SNAPSHOT_CONTENT_CLASS(  # pylint: disable=not-callable
-                    id=snapshot_id, content=snapshot))
+                snapshot_metadata_models.append(
+                    model.SNAPSHOT_METADATA_CLASS.create(
+                        snapshot_id, committer_id, cls._COMMIT_TYPE_DELETE,
+                        commit_message, commit_cmds))
+                snapshot_content_models.append(
+                    model.SNAPSHOT_CONTENT_CLASS.create(
+                        id=snapshot_id, content=snapshot))
```

Comment:
You don't need either of the argnames here since no args are optional.

Reply:
Done.